### PR TITLE
AB#4477 - Changed log messages to show only essentials at success level or higher.

### PR DESCRIPTION
**Pull request by viatunimelb**
[AB#4477](https://dev.azure.com/mduphl/66de697f-df03-4f5d-b5b4-0df4487db257/_workitems/edit/4477)

### CODE REVIEW

**Review by claresloggett — CHANGES_REQUESTED** [review]
The aim here is to use a log level of SUCCESS in our routine logs, right?

The bulk of the messages I see are the "Matched" and "Done" messages, which you've switched to INFO, so those are probably the most important changes for verbosity? Looks good.

For some of the other changes, I personally think you were right the first time, I like the original log levels! I've left a few comments on these. Sorry to be finicky.

File: austrakka/components/sequence/sync/sync_workflow.py [hunk 1]
```diff
@@ -553,7 +565,7 @@ def analyse_status(ctx, use_hash_cache, seq_path, published_manifest):
     p_manifest = published_manifest
 
     if not os.path.exists(seq_path):
-        logger.info(f'Missing: {seq_path}')
+        logger.warning(f'Missing: {seq_path}')
         ctx[DF].at[ctx[IDX], STATUS_KEY] = MISSING
```
[the remark this answers is not in the document]
Comment (viatunimelb) [reply austrakka/components/sequence/sync/sync_workflow.py:569]: It signifies both. For the purpose of self healing, the distinction isn't important.

File: austrakka/components/sequence/sync/sync_workflow.py [hunk 2]
```diff
@@ -378,17 +379,17 @@ def finalise_each_file(int_med, sync_state):
                 int_med.at[index, SAMPLE_NAME_KEY],
                 int_med.at[index, HOT_SWAP_NAME_KEY])
 
-            logger.info(f'Hot swapping: {dest}')
+            logger.warning(f'Hot swapping: {dest}')
             os.rename(src, dest)
-            logger.success(f'Done hot swapping: {dest}')
+            logger.info(f'Done hot swapping: {dest}')
```
[the remark this answers is not in the document]
Comment (viatunimelb) [reply austrakka/components/sequence/sync/sync_workflow.py:385]: Hot swapping is a destructive process, unlike purge which currently just soft-deletes by moving files to trash. The event is worthy of a warning and should appear in logs for traceability.

File: austrakka/components/sequence/sync/sync_workflow.py [hunk 3]
```diff
@@ -123,7 +123,8 @@ def set_state_aggregating(sync_state: dict):
 
 
 def aggregate(sync_state: dict):
-    logger.info(f'Started: {Action.aggregate}')
+    logger.success(f'Started: {Action.aggregate}')
+
```
[the remark this answers is not in the document]
Comment (viatunimelb) [reply austrakka/components/sequence/sync/sync_workflow.py:127]: It's important when the process is killed and then resumes from the beginning of the current action. Logging the start event makes that log trail less confusing.

**Review by claresloggett — APPROVED** [review]

**Pull request merged**